### DISCUSSION TITLE
Export Query

### DISCUSSION THREAD

**Discussion by IvanFarkas** [datalust/seq-tickets#1607] [Help]
### Export Query

I created several queries.
One of them is **Count by Day - sql**
How do I export this query (or all queries) to be able to import it (them) into a different Seq installation, please?

```sql
select count(*) from stream group by time(1d) limit 1000
```
<img width="134" alt="Queries" src="https://user-images.githubusercontent.com/11818001/178825471-c593fe26-11cf-43d9-8213-e29261c3e22b.png">
<img width="170" alt="Query_result" src="https://user-images.githubusercontent.com/11818001/178825675-fe7697e8-a1b7-43e3-9c4a-803674f05c39.png">

**Comment by nblumhardt** [comment]
Thanks for getting in touch, Ivan. I've followed up via your support email. Best regards, Nick.
